server/auth/local.js

```diff
@@ -1,27 +1,41 @@
 const router = require('express').Router();
-const { User } = require('../db/index');
+const { User, Order } = require('../db/index');
+const { findGuestCartMiddleware } = require('../../utils/backend');
 
-router.post('/signup', async (req, res, next) => {
-  try {
-    const user = await User.signup(req.body);
-    req.session.userId = user.id;
-    res.json(user);
-  } catch (error) {
-    next(error);
+router.post(
+  '/signup',
+  findGuestCartMiddleware(Order),
```

Comment:
@justin-cook-developer Could you explain what is happening here? Is the guest cart getting added as a user cart when someone signs up?